README.md

```diff
@@ -11,24 +11,24 @@ natural language processing.
 
 ## Abstract
 Chest X-Rays are considerably reliable radiobiological imprints of patients, which are widely used to diagnose an array 
-of common diseases affecting organs within the chest. For too long, vast accumulations of image data and their 
+of common diseases affecting organs within the chest. Over a period, vast accumulations of image data and their 
```

Comment:
"Over a period" doesn't preserve the original intent. An alternative would be better, else we should maintain "for too long".